### DISCUSSION TITLE
Wix Bundler uninstalls previous version if latest version installation is cancelled from the msi

### DISCUSSION THREAD

**Discussion by H1manshuu** [wixtoolset/issues#8159] [Questions]
I am facing an issue where I want to upgrade two MSIs inside the Wix bundler, upon upgrading if any of the MSI's installation is canceled, then it removes the already installed version as well. I want it to continue upgrading even if one of the MSI's installations is canceled.

Wix Bundler version - 3.11
Wix MSI - 4

Any leads are much appreciated.

**Comment by wmanning** [comment]
I'm not sure if [RollbackBoundary ](https://wixtoolset.org/docs/v3/xsd/wix/rollbackboundary/) would help in that case or not. I never used them but they are suppose to allow packages to stay installed if others fail to install.

**Reply by H1manshuu** [reply]
i tried using RollbackBoundary , it still did not work, any other solution recommended?
